src/main/java/com/sap/cloudfoundry/client/facade/rest/CloudControllerRestClientImpl.java

```diff
@@ -1719,68 +1693,70 @@ private CloudServiceInstance findServiceInstanceByName(String name) {
         return fetchWithAuxiliaryContent(() -> getServiceInstanceResourceByName(name), this::zipWithAuxiliaryServiceInstanceContent);
     }
 
-    private Flux<? extends Resource<UnionServiceInstanceEntity>> getServiceInstanceResources() {
-        IntFunction<ListSpaceServiceInstancesRequest> pageRequestSupplier = page -> ListSpaceServiceInstancesRequest.builder()
-                                                                                                                    .returnUserProvidedServiceInstances(true)
-                                                                                                                    .spaceId(getTargetSpaceGuid().toString())
-                                                                                                                    .page(page)
-                                                                                                                    .build();
+    private Flux<? extends ServiceInstanceResource> getServiceInstanceResources() {
```

Comment:
Isn't `Flux<ServiceInstanceResource>` enough? I think the `? extends` is only needed with `? extends ServiceInstance`